### ISSUE TITLE
More Weights and Italic Options

### ISSUE THREAD

**Issue by oliviamwendt** [graphicore/Questrial-Font#1]
This font is amazing, but very limited due to the nature of only having one weight and no options for italics. Please add more options, it would be a godsend!

**Comment by graphicore** [comment]
Hi Olivia,

thanks for requesting and welcome to GitHub :-)

Unfortunately, I'm not a designer and this repository is only a start of a project, for a designer to pick up from here. It never came to that and as far as I can tell, the files in this repository were not used to even create a (released) font binary.

Additionally, the OFL.txt has the RFN (Reserved Font Name) clause applied, which makes it hard to continue with this project without the designers consent on keeping the name, or the name of the family has to be changed.

If we could fund the further development of this project, I'd be happy to help finding a designer and support on the technical aspects. I can also ask around a bit but I can't promise anything.

I tried but I couldn't find current online traces of the original designer Joe Prince.

**Comment by oliviamwendt** [comment]
Hey,

Thanks for the quick reply! 

Is this something I could personally pursue creating modified versions of?

I'm assuming that I would probably have to change the name. I have a font programming software and I am able to modify the original in vector format.  I am just not 100% on the legalities of this. 

Let me know! :)

**Comment by graphicore** [comment]
Yes, totally. You should maybe read the license. https://github.com/graphicore/Questrial-Font/blob/master/OFL.txt or at least about the license https://en.wikipedia.org/wiki/SIL_Open_Font_License it's a [copyleft](https://en.wikipedia.org/wiki/Copyleft) license, which means you have to keep the OFL for your derivative work. However, the RFN means you can't keep the original name. I strongly suggest to not use the RFN in a derivative work, as it's more annoying than doing good in a libre font project.

Here's a quote about the RFN from (the Google Fonts Project Checklist)[https://github.com/googlefonts/gf-docs/blob/ab486a51d2457ede33023ef4bab264eeaa0bfd6c/ProjectChecklist/README.md]:

> The OFL Reserved Font Name (RFN) option is not recommended for 2 reasons:
>
>1. It makes the process of collaboration via github cumbersome;
if a memberof the general public clicks the 'fork' button up top right, they must either seek permission from you before-hand to use the RFN, or to immediately change the font name in the files as part of their first modification, and then you'll have to change it back.
>
> 2. Many websites who self host the font files will modify them, doing subsetting and format conversion, which are both explicitly defined in the OFL as kinds of modification subject to the RFN restriction.
> SIL have been willing to offer their view that the most popular web font formats are not format conversions subject to the RFN restriction, but popular web font bundling service Font Squirrel includes the less popular formats too, which do.
And subsetting remains.
So, changing the name or contacting you are both administrative burdens for these downstream users, and if they do contact you, for you;
and many people won't think carefully about this and will just do so without asking, which formally terminates their license.
Putting them in that position is not a good situation.
>
> So not having any RFNs is best, unless you have a specific reason to use it.
>
>For more discussion of this topic, see
>
> * <https://lwn.net/Articles/552178>
> * <https://github.com/simoncozens/silson/issues/1>

**Comment by graphicore** [comment]
In general that [Project Checklist](https://github.com/googlefonts/gf-docs/blob/master/ProjectChecklist/README.md) should be a good starting point.

**Comment by davelab6** [comment]
@oliviamwendt did you get anywhere with expanding this project? :)

**Comment by oliviamwendt** [comment]
Hey Dave!
I have a bold and a thin weight that I have the characters for, but I haven’t completed it due to the fact that the kerning is such a huge task.
If you are interested in working on the kerning I can send you the files. 
Currently using Fontself as the program for this.
How did you become interested in this project ?

Sent from Yahoo Mail for iPhone


On Monday, June 15, 2020, 4:58 PM, Dave Crossland <notifications@github.com> wrote:




@oliviamwendt did you get anywhere with expanding this project? :)
